### PR TITLE
New App: Ski Report for Epic Resort Mountains

### PR DESCRIPTION
**Pull request by cmorrisseau77**
The Ski report app can report the number of open trails, weather and 24 hour snowfall for any of the 30+ North American Ski resorts running on Epic Pass
ski_report

### CODE REVIEW

**Comment by cmorrisseau77** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by cmorrisseau77** [comment]
recheck

File: apps/skireport/ski_report.star [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    Args:
+        resort (string): The Resort Name as listed in RESORT_URLS
+
+    Returns:
+        _type_: _description_
+    """
+    url = RESORT_URLS[resort] + TERRAIN_URL_STUB
+
+    # Pull an HTML response of the lift status page
+    r = http.get(url)
```
Comment (matslina) [inline apps/skireport/ski_report.star:136]: We're going to need to add some caching around the http calls to make sure request volume isn't too great.

**Review by matslina — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: apps/skireport/ski_report.star (hunk 1 above)
Comment (cmorrisseau77) [reply apps/skireport/ski_report.star:145]: I've added caching every ten minutes for each call in the latest commit

**Review by matslina — APPROVED** [review]

**Comment by matslina** [comment]
Thanks! Looks like there are some formatting changes required. Running make format (or applying the suggest changes manually) should hopefully do the trick.

**Comment by cmorrisseau77** [comment]
Oops. I Ran Make Format the first time and forgot to run it after the changes

File: apps/skireport/ski_report.star [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+def get_schema():
+    options = [schema.Option(display = resort, value = resort) for resort in RESORT_URLS.keys()]
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Dropdown(
+                id = "resort",
+                name = "Ski Resort",
+                desc = "The resort you want to show. North American Epic Pass Resorts only right now",
```
Comment (matslina) [inline apps/skireport/ski_report.star:75]:
```suggestion
                icon = "mountain",
                desc = "The resort you want to show. North American Epic Pass Resorts only right now",
```

**Pull request merged**